appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -194,6 +196,11 @@ private void callDelegateWithOneError(
     byte[] result = null;
     final Double timeoutInSeconds = 10.0;
 
+    Logger logger = Logger.getLogger(VmApiProxyDelegate.class.getName());
+    Level level = logger.getLevel();
+    if (!logger.isLoggable(Level.FINE)) {
+      logger.setLevel(Level.SEVERE);
```

Comment:
I am not sure why we jump from `FINE` to `SEVERE`. If we want to suppress anything bellow `SEVERE` we should check for `isLoggable(Leve.WARNING)` or if we only care to suppress `FINE` and bellow we should set it to `CONFIG`.